documentation/aks-upgrade.md

```diff
@@ -84,7 +84,7 @@ az aks nodepool get-upgrades --resource-group <ResourceGroup> --cluster-name <Cl
 ## Upgrade process
 
 1. Determine which is the next available version (see above)
-1. Test the whole process in a dev cluster:
+2. Test the whole process in a dev cluster:
```

Comment:
Markdown automatically increases the numbering if all lines have "1.". Can we keep it as it is?